modules/kmm-configuring-the-lookup-path-on-nodes.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-configuring-the-lookup-path-on-nodes_{context}"]
+= Configuring the lookup path on nodes
+
+On OpenShift nodes, the set of default lookup paths for firmwares does not include `/var/lib/firmware`.
+
+.Procedure
+
+. Use the Machine Config Operator to create a `MachineConfig` resource that contains the `/var/lib/firmware` path:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
```

Comment:
Maybe add a note that the label is user configurable based on his needs . In case of SNOs it should be `control-plane` or `master`

Reply:
/Added a callout: **<1> You can configure the label based on your needs. In the case of SNOs, use either `control-pane` or `master`.**